docs/features/walletconnect.md

```diff
@@ -0,0 +1,106 @@
+
+## How it works
+
+Communication between the dApp and the mobile wallet (_peers_) happens over a _bridge server_ that relays messages without access to their content. Message contents are symmetrically encrypted through a shared key between the two peers using session data. The connection is initiated by the dApp displaying a QR Code or deep link with a standard WalletConnect URI and is established when the user approves this connection request within their Algorand Wallet. Public bridge servers are maintained by WalletConnect and dApp developers may also deploy their own.
+
+Information exchange between the peers typically consists of an unsigned atomic transaction group from the dApp to the mobile wallet, which is then inspected by the user, signed and returned back to the dApp. The dApp will submit the signed object for confirmation by the Algorand network which will effect balances and state accordingly.
+
+<center>![WalletConnect Header](../imgs/walletconnect-establishing-connection.png)</center>
+
+## Quick start for dApp devs
+
+The TypeScript sample code below should allow a dApp developer to establish a connection and request a simple payment transaction for signature. A [detailed example](https://github.com/jasonpaulos/algorand-walletconnect-example-dapp) is offered in React and a [live demo](https://jasonpaulos.github.io/algorand-walletconnect-example-dapp/) is available on TestNet.
+
+### Install
+
+```bash tab="yarn"
+yarn add @walletconnect/client @walletconnect/qrcode-modal @algorand/js-algorand-sdk
+```
+
+```bash tab="npm"
+npm install --save @walletconnect/client @walletconnect/qrcode-modal @algorand/js-algorand-sdk
+```
+
+### Initiate Connection
+
+```typescript
+import WalletConnect from "@walletconnect/client";
+import QRCodeModal from "@walletconnect/qrcode-modal";
+import algosdk from "@algorand/js-algorand-sdk";
+
+// Create a connector
+const connector = new WalletConnect({
+  bridge: "https://bridge.walletconnect.org", // Required
+  qrcodeModal: QRCodeModal,
+});
+
+// Check if connection is already established
+if (!connector.connected) {
+  // create new session
+  connector.createSession();
+}
+
+// Subscribe to connection events
+connector.on("connect", (error, payload) => {
+  if (error) {
+    throw error;
+  }
+
+  // Get provided accounts and chainId
+  const { accounts, chainId } = payload.params[0];
+});
+
+connector.on("session_update", (error, payload) => {
+  if (error) {
+    throw error;
+  }
+
+  // Get updated accounts and chainId
+  const { accounts, chainId } = payload.params[0];
+});
+
+connector.on("disconnect", (error, payload) => {
+  if (error) {
+    throw error;
+  }
+
+  // Delete connector
```

Comment:
since there is already an example of how to create the session, maybe it makes sense to also include a call to `killSession` here to capture an entire lifecycle of `connector` instance in the example?